best-practices-cel/disallow-latest-tag/disallow-latest-tag.yaml

```diff
@@ -24,20 +25,18 @@ spec:
           kinds:
           - Pod
     validate:
-      message: "An image tag is required."
-      pattern:
-        spec:
-          containers:
-          - image: "*:*"
+      cel:
+        expressions:
+          - expression: "object.spec.containers.all(container, container.image.contains(':'))"
+            message: "An image tag is required."
   - name: validate-image-tag
     match:
       any:
       - resources:
           kinds:
           - Pod
     validate:
-      message: "Using a mutable image tag e.g. 'latest' is not allowed."
-      pattern:
-        spec:
-          containers:
-          - image: "!*:latest"
\ No newline at end of file
+      cel:
+        expressions:
+          - expression: "object.spec.containers.all(container, !container.image.endsWith(':latest'))"
+            message: "Using a mutable image tag e.g. 'latest' is not allowed."
```

Comment:
Since VAPs can only be generated from policies with a single rule, would it be a good idea to combine both expressions under one rule as both the rules match on pods?